src/messages/MessageBuilder.hpp

```diff
@@ -60,7 +62,7 @@ class MessageBuilder
     void addLink(const QString &origLink, const QString &matchedLink);
 
     template <typename T, typename... Args>
-    T *emplace(Args &&... args)
+    T *emplace(Args &&...args)
```

Comment:
```suggestion
    T *emplace(Args &&... args)
```

to appease the linter